configuration/analyzer_config.json

```diff
@@ -1878,6 +1878,31 @@
     "secrets": {},
     "params": {}
   },
+  "Phishstats": {
+    "type": "observable",
+    "python_module": "phishstats.PhishStats",
+    "description": "check if the observable is a phishing site",
```

Comment:
Change to:

`Search [PhishStats API](https://phishstats.info/) to determine if an IP/URL/domain is malicious.`

make same change in the Usage.md file.

Reply:
done